### ISSUE TITLE
Warnings

### ISSUE THREAD

**Issue by santi-h** [nickstanish/reprogressbars#4]
Works but I'm getting
<img width="1439" alt="screen shot 2017-02-07 at 10 16 08 pm" src="https://cloud.githubusercontent.com/assets/876161/22723354/2458d6b2-ed83-11e6-87a1-62376f83e7e5.png">

When used like this
```javascript
import React, { Component } from 'react';
import NavBar from './nav_bar';
import { ProgressBar } from 'reprogressbars';

export default (props) => (
  <div>
    <NavBar />
    <ProgressBar isLoading={true} color="#33a0c6" height="2px"/>
    {props.children}
  </div>
)
```

Or like this

```javascript
import React, { Component } from 'react';
import NavBar from './nav_bar';
import { ProgressBar } from 'reprogressbars';

export default class ApplicationWrapper extends Component {
  constructor(props) {
    super(props);
    this.state = {};
  }

  render() {
    return (
      <div>
        <NavBar />
        <ProgressBar isLoading={true} color="#33a0c6" height="2px"/>
        {this.props.children}
      </div>
    );
  }
}
```

**Comment by nickstanish** [comment]
This is due to starting the animation before the component is mounted. Will create a fix soon

**Comment by nickstanish** [comment]
Fixed in 0.2.1
